cmd/watch.go

```diff
@@ -53,7 +52,7 @@ func (w *watchRuns) run(event filewatcher.Event) error {
 	opts.packages = append(opts.packages, event.Args...)
 
 	var err error
-	if w.prevExec, err = runSingle(&opts, dir); !IsExitCoder(err) {
+	if w.prevExec, err = runSingle(w.ctx, w.cancel, &opts, dir); !IsExitCoder(err) {
```

Comment:
If you need to pass a context here it means the method you are in should receive it.